app/views/partials/activity_details_events_table.scala.html

```diff
@@ -132,6 +132,14 @@
                                 useTabIndex = Some(startAtTabIndex + (index*2))
                             )
                         }
+
+                        case EventType.CASE_CREATED => {
+                            val body = event.details.asInstanceOf[CaseCreated]
+                                <div id="activity-events-row-case_created">
+                                <span id="activity-events-row-application_submitted-comment" class="pre-wrap">{body.comment}</span>
+                                </div>
```

Comment:
Should we use the `progressive_disclosure` partial here in order to preserve the tabindex functionality? Or even define a new partial `activity_case_created` similar to `activity_case_completed`?

Reply:
Good point, it now uses progressive_disclosure 👍